pootle/core/utils/version.py

```diff
@@ -183,6 +183,29 @@ def _shell_command(command):
     return command_subprocess.communicate()[0]
 
 
+def _is_executable_file(path):
+    return os.path.isfile(path) and os.access(path, os.X_OK)
+
+
+def _git_binary():
+    """
+    Tries to find the path to the `git` program.
+    Can be overridden by setting POOTLE_GIT_PATH environment variable.
+    See #3876.
+    """
+    setting = os.environ.get("POOTLE_GIT_PATH")
```

Comment:
Is this really necessary to add a new custom `POOTLE_GIT_PATH` envvar? IMHO it is good enough to use the logic below.